docs/csharp/write-safe-efficient-code.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: Write safe and efficient C# code
 description: Recent enhancements to the C# language enable you to write verifiable safe code that the performance previously associated with unsafe code. 
-ms.date: 10/23/2018
+ms.date: 03/17/2020
```

Comment:
That's not a typo. A new date is based on the latest significant update of the article: #17487